src/inputHandler.ts

```diff
@@ -128,4 +133,16 @@ export class InputHandler {
                 return;
           }
     }
+
+    private updateHunterSpeed(sliderValue: number, sliderMin: number, sliderMax: number) {

```

Comment:
Aren't the min and max numbers always 0 and 100? If so, we shouldn't need to pass in those values here.